physionet-django/user/views.py

```diff
@@ -573,12 +573,16 @@ def edit_credentialing(request):
 
 
 @login_required
-def user_credential_applications(request):
+def user_credential_applications(request, user=None):
```

Comment:
I think it would be clearer to use `username` as the argument, if this is what you are expecting to receive.